client/web/src/insights/components/context-menu/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { getCollisions, Position } from '@reach/popover'
+
+const DEFAULT_VERTICAL_OFFSET = 4
+
+/**
+ * Custom popover position calculator. Returns position objects (top,left,right,bottom) styles
+ * with values such that the target and the popover element have the same right borders.
+ *
+ * ------------ | Target | --------
+ * ----|*****************| --------
+ * ----|*****************| --------
+ * ----|*** Popover *****| --------
+ * ----|*****************| --------
+ * ----|*****************| --------
```

Comment:
Indenting by 4 spaces makes it a code block, to make sure the diagram is not mangled in hover tooltips :)

![image](https://user-images.githubusercontent.com/10532611/126754166-14b9ee1c-6c32-4755-bd26-e1e8b8251979.png)


```suggestion
 *     ------------ | Target | --------
 *     ----|*****************| --------
 *     ----|*****************| --------
 *     ----|*** Popover *****| --------
 *     ----|*****************| --------
 *     ----|*****************| --------
```

Reply:
@felixfbecker thanks for catching this. I ended up with pre tag for it since 4 spaces way doesn't work with Webstorm and we have eslint for the jsdoc that requires no indentation (`There must be no indentation.(jsdoc/check-indentation)`)

With pre tag
<img width="752" alt="Screenshot 2021-07-23 at 11 45 50" src="https://user-images.githubusercontent.com/18492575/126758665-0aec3e47-e6a6-4379-a00f-dbdc279bfd5f.png">

<img width="584" alt="Screenshot 2021-07-23 at 11 47 10" src="https://user-images.githubusercontent.com/18492575/126758679-b08f4e88-5748-470f-b72a-5a1d45b84abc.png">

Reply:
You can also use three backticks around it :)

Reply:
Actually, lol, it looks like without backticks and only `<pre>` the asterisks in your second screenshot still get interpreted as markdwon, which makes the "Popover" text italic and some of the text bold 😆